src/Backend/Modules/Search/Installer/Data/install.sql

```diff
@@ -26,14 +24,13 @@ CREATE TABLE IF NOT EXISTS `search_statistics` (
  `data` text,
  `num_results` int(11) default NULL,
  PRIMARY KEY (`id`)
-) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci AUTO_INCREMENT=1 ;
-
+) ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci AUTO_INCREMENT=1 ;
 
 CREATE TABLE IF NOT EXISTS `search_synonyms` (
  `id` int(11) NOT NULL auto_increment,
- `term` varchar(255) NOT NULL,
+ `term` varchar(245) NOT NULL,
```

Comment:
I don't think this will cause any issues.

I would merge this in the 3.10 branch though, since it is backwards compatibility breaking (mysql tables change).

Could you change the target of the pull request?